x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -166,7 +167,40 @@ export async function getKibanaAssets(
   return result;
 }
 
-async function installKibanaSavedObjects({
+const isImportConflictError = (e: SavedObjectsImportFailure) => e?.error?.type === 'conflict';
+/**
+ * retry saved object import if only conflict errors are encountered
+ */
+async function retryImportOnConflictError(
+  importCall: () => ReturnType<SavedObjectsImporterContract['import']>,
+  {
+    logger,
+    maxAttempts = 50,
+    _attempt = 0,
+  }: { logger?: Logger; _attempt?: number; maxAttempts?: number } = {}
+): ReturnType<SavedObjectsImporterContract['import']> {
+  const result = await importCall();
+
+  const errors = result.errors ?? [];
+  if (_attempt < maxAttempts && errors.length && errors.every(isImportConflictError)) {
+    const retryCount = _attempt + 1;
+    const retryDelaySec = 1 + Math.floor(Math.random() * 3); // 1s + 0-3s of jitter
```

Comment:
nit: let's use milliseconds instead of seconds to increase the resolution of the randomness of the jitter a bit (so there will be values like 1523 and 24 and not just 1 and 0)

```suggestion
    const retryDelayMs = 1000 + Math.floor(Math.random() * 3000); // 1s + 0-3s of jitter
```